src/components/views/messages/MVideoBody.tsx

```diff
@@ -228,6 +228,18 @@ export default class MVideoBody extends React.PureComponent<IBodyProps, IState>
         const content = this.props.mxEvent.getContent();
         const autoplay = SettingsStore.getValue("autoplayVideo");
 
+        let aspectRatio;
+        if (content.info?.w && content.info?.h) {
+            aspectRatio = content.info.w / content.info.h;
+        }
+        const { w: maxWidth, h: maxHeight } = suggestedVideoSize(
+            SettingsStore.getValue("Images.size") as ImageSize,
+            { w: content.info?.w, h: content.info?.h },
+        );
+
+        // HACK: This div fills out space while the video loads, to prevent scroll jumps
+        const spaceFiller = <div style={{ width: maxWidth, height: maxHeight }} />;
```

Comment:
What hides this? it seems to always get rendered with both the spinner and video tags unlike in MImageBody?

Does it just get pushed off by the video/spinner and cut by the overflow none? Can we do a similar state.loaded solution to not pollute the DOM too much?

Reply:
Yep, it currently gets cut off by overflow, will look into hiding it on load

Reply:
I tried this, but couldn't get any of `onLoad`, `onLoadedMetadata`, `onLoadedData` to fire, so let's live with this bit of DOM pollution for now.